src/Extensions.Hosting/HostBuilderExtensions.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Diagnostics;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Json;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+using Serilog;
+
+namespace Bit.Extensions.Hosting;
+
+public static class HostBuilderExtensions
```

Comment:
❓ In my PR I lifted some different app config and assemble extensions too -- what should be picked?

Reply:
Yeah that is probably a good thing to lift to this project.